src/plugins/tscn/tscnplugin.cpp

```diff
@@ -693,6 +774,35 @@ bool TscnPlugin::write(const Map *map, const QString &fileName, Options options)
 
             layerIndex++;
         }
+
+        device->write("\n");
+                    
+        // Object scene nodes
+        for (const MapObject *object : assetInfo.objects) {
+            device->write("\n");
+
+            auto name = object->name();
+            if (name.isEmpty())
+                name = "Object" + QString::number(object->id());
+
+            auto resPath = object->resolvedProperty("resPath").toString();
+
+            device->write(formatByteString(
+                "[node name=\"%1\" parent=\".\" instance=ExtResource(\"%2\")]\n",
+                sanitizeQuotedString(name),
+                sanitizeQuotedString(assetInfo.objectIds[resPath]))
+            );
+
+            auto pos = object->position();
+            pos.setX(pos.x() + object->resolvedProperty("originX").toFloat());
+            pos.setY(pos.y() + object->resolvedProperty("originY").toFloat());
```

Comment:
It seems like it would make sense to take into account the "Object Alignment" tileset property here, which provides an easier way of changing the object origin (for tile objects), which can also keep the position of the object in Tiled the same as the position of the object in Godot.

"Object Alignment" can be used when one of its values matches the origin in Godot, and all objects in the same tileset share the same alignment.

Reply:
Much simpler! (The workflow; not the code ;) )

Done.